en/pe-docs/docs/devops-and-ci-cd/manage-environments.md

```diff
@@ -0,0 +1,44 @@
+# Manage Environments
+
+By default, all projects created in the cloud data planes (irrespective of the data plane region) are provisioned with two environments (i.e., development and production).
+
+The environments are listed in the order of deployment and promotion. The initial deployment takes place in the first environment and you can proceed to promote a component to subsequent environments.
+
+## Create a new environment
+
+### Prerequisites
+
+- To create additional environments, you must have a Choreo subscription or a private data plane.
+- To create a new environment in a private data plane organization, you must have the `ENVIRONMENT-MANAGEMENT` permission. By default, `ENVIRONMENT-MANAGEMENT` permission is granted to Admin and Choreo DevOps roles.
+
+To create a new environment, follow the steps given below:
+
+1. Sign in to the [Choreo Console](https://console.choreo.dev/) and select the **Organization**.
+2. In the left navigation menu, click **Infrastructure** and then click **Environments** (note that this is the **Environments** page under your organization, not your projects).
+3. On the **Environments** page, click **Create** and specify the following details to create a new environment:
+   
+    - **Name**: A display name for the new environment.
+    - **Data Plane** - The data plane to create the new environment.
+
+        !!!tip
+            The **Data Plane** list displays all the private data planes registered under your organization. 
```

Comment:
```suggestion
            The **Data Plane** list displays all the private data planes registered under your organization along with Choreo cloud data planes.
```

Reply:
Have to check if this the case

Reply:
<img width="1717" alt="Screenshot 2025-03-11 at 08 45 55" src="https://github.com/user-attachments/assets/7145c488-882b-4e90-b8df-ec404b6fde6b" />

Reply:
Unfortunately I do not have this pdp running, hence it does not show in creation view.

Reply:
raveen confirmed that it is valid. Both DPs are available

Reply:
Here the listing happens like this:

For CDP free users:
They will see the CDP they are in(the one they have selected in org initialisation)

For CDP paid Users:
If they have created environments in both CDPs, they. will see both here
If not, they will only see whatever the CDP they have environments created in

For PDP Only users:
They will see their PDP only

For Hybrid Users(One click PDPs + CDPs)
They will see all the dataplanes they have including PDPs and CDPs they have environments in

So in short, what you see here are the consumed dataplanes of the user.

So I would say the line as below:
`The **Data Plane** list displays all the private data planes registered under your organization along with **consumed** Choreo cloud data planes.`